Formula/ncurses.rb

```diff
@@ -31,7 +31,9 @@ def install
                           "--enable-widec",
                           "--with-shared",
                           "--with-gpm=no",
-                          "--without-ada"
+                          "--without-ada",
+                          # Avoid spurious linkage against system terminfo directories
+                          "--with-terminfo-dirs=''"
```

Comment:
Can we scope this change to Linux? macOS does ship with its own `terminfo` database, and it's not clear what this flag changes about that.

Reply:
Actually, I feel like this breaks `ncurses`? The `INSTALL` docs say
```
    --with-terminfo-dirs=XXX
	Specify a search-list of terminfo directories which will be compiled
	into the ncurses library (default: DATADIR/terminfo)

	This is a colon-separated list, like the TERMINFO_DIRS environment
	variable.
```
which means that `ncurses` will need additional post-install configuration to even find its own `terminfo` directories, which isn't what we want.